src/openforms/config/views.py

```diff
@@ -107,19 +107,12 @@ def get_plugin_entry(self, plugin: Any) -> Entry:
 
         try:
             plugin.check_config()
-        except InvalidPluginConfiguration as e:
-            status_message = e
-            status = False
-        except NotImplementedError:
-            status_message = _("Not implemented")
-            status = None
         except Exception as e:
-            status_message = _("Internal error: {exception}").format(exception=e)
-            status = False
-        else:
-            status_message = None
-            status = True
-
+            status_message, status, error = (
+                _("Failed to validate the configuration."),
+                False,
+                e,
```

Comment:
**Mistakes**

* `e` is of `Exception` type, while the type hint expects `str` type